### PR TITLE
Feat/hub error

### PR DESCRIPTION
**Pull request by marie-j**

### CODE REVIEW

**Comment by FredericEspiau** [comment]
No ticket reference @marie-j  ? 😢

File: packages/manager/modules/hub/src/components/payment-status-tile/payment-status-tile.html [hunk 1]
```diff
@@ -5,10 +5,20 @@
     data-href="{{ ::$ctrl.autorenewLink }}"
     data-on-click="$ctrl.onLinkClick()"
 >
-    <div class="oui-datagrid-responsive-container">
+    <hub-tile-error
+        ng-if="!$ctrl.services && !$ctrl.loading"
+        message="{{ 'ovh_manager_hub_payment_status_tile_error' | translate }}"
```
Comment (varun257) [inline packages/manager/modules/hub/src/components/payment-status-tile/payment-status-tile.html]: `message="{{:: 'ovh_manager_hub_payment_status_tile_error' | translate }}"`

File: packages/manager/apps/hub/src/dashboard/dashboard.component.js [hunk 2]
```diff
@@ -7,6 +7,9 @@ export default {
     billingServices: '<',
     notifications: '<',
     order: '<',
+    refreshBillingServices: '<',
+    refreshOrder: '<',
+    refresh: '<',
```
Comment (radireddy) [inline packages/manager/apps/hub/src/dashboard/dashboard.component.js]: sort please

File: packages/manager/apps/hub/src/dashboard/dashboard.html [hunk 3]
```diff
@@ -9,15 +9,20 @@ <h2 data-translate="manager_hub_dashboard_overview"></h2>
 <div class="row">
     <div class="col-md-8">
         <hub-payment-status
-            data-services="$ctrl.billingServices.data"
-            data-tracking-prefix="{{ $ctrl.trackingPrefix }}"
-            data-total-count="$ctrl.billingServices.count"
+            services="$ctrl.billingServices.data"
```
Comment (radireddy) [inline packages/manager/apps/hub/src/dashboard/dashboard.html]:
```suggestion
            data-services="$ctrl.billingServices.data"
```

File: packages/manager/modules/hub/src/components/tile-error/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+  "manager_hub_error_tile_oops": "Oops ...!",
```
Comment (FredericEspiau) [inline packages/manager/modules/hub/src/components/tile-error/translations/Messages_fr_FR.json]: 😢

File: packages/manager/apps/hub/src/dashboard/dashboard.html [hunk 5]
```diff
@@ -9,15 +9,20 @@ <h2 data-translate="manager_hub_dashboard_overview"></h2>
...
+            tracking-prefix="{{ $ctrl.trackingPrefix }}"
+            total-count="$ctrl.billingServices.count"
+            refresh="$ctrl.refreshBillingServices()"
```
Comment (radireddy) [inline packages/manager/apps/hub/src/dashboard/dashboard.html]: prefix with data- for all custom attributes

File: packages/manager/apps/hub/src/dashboard/dashboard.html [hunk 6]
```diff
@@ -9,15 +9,20 @@ <h2 data-translate="manager_hub_dashboard_overview"></h2>
...
         >
         </hub-payment-status>
     </div>
     <div class="col-md-4">
-        <hub-billing-summary bills="$ctrl.bills"></hub-billing-summary>
-        <hub-order-tracking order="$ctrl.order"></hub-order-tracking>
+        <hub-billing-summary
+            bills="$ctrl.bills"
+            refresh="$ctrl.refresh('bills')"
```
Comment (radireddy) [inline packages/manager/apps/hub/src/dashboard/dashboard.html]: data-

File: packages/manager/modules/hub/src/components/payment-status-tile/index.js [hunk 7]
```diff
@@ -5,14 +5,21 @@ import 'ovh-ui-angular';
...
 import './payment-status-tile.less';
 
 const moduleName = 'ovhManagerHubPaymentStatusTile';
 
 angular
-  .module(moduleName, [ngAtInternet, 'ngTranslateAsyncLoader', hubTile, 'oui'])
+  .module(moduleName, [
+    ngAtInternet,
+    'ngTranslateAsyncLoader',
+    hubTile,
```
Comment (FredericEspiau) [inline packages/manager/modules/hub/src/components/payment-status-tile/index.js]: nit: order

File: packages/manager/modules/hub/src/components/order-tracking/template.html [hunk 8]
```diff
@@ -1,35 +1,43 @@
...
-        <span
-            class="oui-icon oui-icon-arrow-right_extra-thin"
-            aria-hidden="true"
+        <div class="mb-3">
+            <span
+                class="font-weight-bold"
+                data-ng-bind="$ctrl.currentStatus.date | date:'shortDate'"
+            ></span>
+            <span
+                data-translate="{{ 'order_tracking_history_' + $ctrl.currentStatus.label }}"
```
Comment (radireddy) [inline packages/manager/modules/hub/src/components/order-tracking/template.html]: may be one-time binding can be used here?

File: packages/manager/modules/hub/src/components/payment-status-tile/payment-status-tile.html (hunk 1 above)
Comment (radireddy) [inline packages/manager/modules/hub/src/components/payment-status-tile/payment-status-tile.html]: data- and one time binding

File: packages/manager/apps/hub/src/dashboard/dashboard.html (hunk 5 above)
Comment (marie-j) [reply packages/manager/apps/hub/src/dashboard/dashboard.html]: @radireddy Not mandatory as those are custom components

**Comment by JeremyDec** [comment]
@marie-j you have two commits with the same name ?

**Comment by marie-j** [comment]
> @marie-j you have two commits with the same name ?

@JeremyDec No 
One is  `add error TEMPLATE for tiles` the other one is `add error HANDLING for tiles`
I'll try to find something maybe less confusing

File: packages/manager/modules/hub/src/components/billing-summary/controller.js [hunk 9]
```diff
@@ -19,6 +20,9 @@ export default class ManagerHubBillingSummaryCtrl {
           get(data, 'currency.code'),
         );
         return this.bills;
+      })
+      .finally(() => {
+        this.loading = false;
```
Comment (JeremyDec) [inline packages/manager/modules/hub/src/components/billing-summary/controller.js]: `loading` only concerns bills loading ? Not loading debt also ?

File: packages/manager/modules/hub/src/components/billing-summary/controller.js [hunk 10]
```diff
@@ -82,4 +86,15 @@ export default class ManagerHubBillingSummaryCtrl {
       .then(({ data }) => data)
       .then(({ data }) => data.debt);
   }
+
+  refreshTile() {
+    this.loading = true;
+    this.refresh()
```
Comment (JeremyDec) [inline packages/manager/modules/hub/src/components/billing-summary/controller.js]: ?
```suggestion
    return this.refresh()
```

**Comment by marie-j** [comment]
> No ticket reference @marie-j ? 😢

@FredericEspiau was not originally planned but will be tested at the same time than the dashboard

File: packages/manager/modules/hub/src/components/billing-summary/controller.js (hunk 9 above)
Comment (marie-j) [reply packages/manager/modules/hub/src/components/billing-summary/controller.js]: You're right, loading was not put at the the right place

**Comment by JeremyDec** [comment]
@marie-j LGTM, but you have conflicts :)

**Comment by marie-j** [comment]
Solved conflicts

**Pull request merged**